test/storage/garbage_collector_test.cpp

```diff
@@ -1457,16 +1471,16 @@ TEST_F(GarbageCollectorTests, DeleteUpdateTest3) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```